src/pages/workspaces/workspace/analysis/modals/ComputeModal.js

```diff
@@ -1321,7 +1321,7 @@ export const ComputeModalBase = ({
             div({ style: { margin: '1rem 0 0.5rem', fontSize: 16, fontWeight: 600 } }, ['What would you like to do with your disk?']),
             DeleteDiskChoices({
               existingEnvironmentConfig: getExistingEnvironmentConfig(),
-              persistentDiskCost: Utils.formatUSD(getPersistentDiskCostMonthly(currentPersistentDiskDetails, computeConfig.computeRegion)),
+              persistentDiskCostLabel: Utils.formatUSD(getPersistentDiskCostMonthly(currentPersistentDiskDetails, computeConfig.computeRegion)),
```

Comment:
So... "Label" isn't really what this field is either.
If you want to keep it as is though, that's fine.